src/main/java/Task.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Creates a task with a description which is to be added
```

Comment:
This describes the constructor, not the class.

Reply:
Agreed. A tendency of the characteristics of a javadocs for a method or constructor